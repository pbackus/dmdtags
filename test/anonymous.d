enum
{
	int x
}

struct S
{
	enum
	{
		int y
	}
}
