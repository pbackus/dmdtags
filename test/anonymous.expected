!_TAG_FILE_SORTED	1	/0=unsorted, 1=sorted, 2=foldcase/
S	test/anonymous.d	6;"	s
anonymous	test/anonymous.d	1;"	M
x	test/anonymous.d	3;"	e
y	test/anonymous.d	10;"	e	struct:S
